libs/remix-ui/terminal/src/lib/remix-ui-terminal.tsx

```diff
@@ -87,7 +87,13 @@ export const RemixUiTerminal = (props: RemixUiTerminalProps) => {
       },
 
       log: (message) => {
-        scriptRunnerDispatch({ type: 'log', payload: { message: [message] } })
+        if (typeof message === 'string') {
+          message = {
+            value: message,
+            type: 'info'
```

Comment:
This may be better to keep the type as `log` as `info` ensure a green/success message on terminal